docs/sources/mimir/release-notes/v2.9.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Grafana Mimir version 2.9 release notes"
+menuTitle: "V2.9 release notes"
+description: "Release notes for Grafana Mimir version 2.9"
+weight: 300
+---
+
+# Grafana Mimir version 2.9 release notes
+
+Grafana Labs is excited to announce version 2.9 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Experimental support for optimized series selection strategy** Mimir can now use an optimized strategy for fetching posting lists that can significantly reduce memory allocations in the store-gateway.
```

Comment:
This is a great optimization but this description feels a little dense for release notes. Maybe @dimitarvdimitrov can distill it down?

Reply:
```suggestion
- **Experimental support for optimized series selection strategy** The store-gateway can now minimize the data necessary to complete requests. Based on the query it can fetch slightly more series and filter them down them before returning as opposed to spending resources to narrow down the selection beforehand.
```

How about this? Alternatively we can just say that the store-gateway can now complete requests with fetching less data.
```suggestion
- **Experimental support for optimized series selection strategy** Reduce the volume of data necessary to complete requests with individual label selectors that are present on a lot of series.
```

I think I prefer the latter.